oximeter/oximeter/schema/hardware-component.toml

```diff
@@ -181,3 +182,21 @@ datum_type = "cumulative_u64"
 versions = [
     { added_in = 1, fields = ["error"] }
 ]
+
+[[metrics]]
+name = "cpu_tctl"
```

Comment:
I wondered if we should maybe be calling this `amd_cpu_tctl` or similar, since T<sub>ctl</sub> is an AMD-specific value, and hypothetical sleds with non-AMD CPUs may have different thermal readings, like the similar T<sub>jMAX</sub> on Intel processors. 

cc @rmustacc, do you have any opinions on naming here?

Reply:
I suppose another option would be to give the metric a more generic name such as `cpu_thermal`, and stick the specific thing in a field  like `kind = "amd_tctl"` or something. How much do you think we should be thinking about future-proofing this schema for possible CPUs that aren't broadly AMD Zen-like?

Reply:
Yup, per our conversation this morning, it seems like the main reason to make things the same metric is so that they can be *grouped*, not because it's desirable to avoid having targets that don't contain the same set of metrics. I'm going to change it to `amd_cpu_tctl` though just to make it clearer to the reader what this is.